src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -105,7 +109,7 @@ StringSubject diagnosticStream() {
       roots.add("main.js");
     }
 
-    List<SourceFile> externFiles = new ArrayList<>();
+    List<SourceFile> externFiles;
```

Comment:
yep, unrelated clean-up. Intellij made is different color, so hard to resists :)